kitsune/users/tests/test_views.py

```diff
@@ -442,3 +448,37 @@ def test_id_token(self, key):
         )
         self.assertEqual(400, response.status_code)
         self.assertEqual(0, AccountEvent.objects.count())
+
+
+class UserCloseAccountTests(TestCase):
+    def setUp(self):
+        self.user = UserFactory(
+            username="ringo", email="ringo@beatles.com", groups=[GroupFactory()]
+        )
+        self.client.login(username=self.user.username, password="testpass")
+        super(UserCloseAccountTests, self).setUp()
+
+    def test_close_account(self):
+        """Test the closing of a user's account."""
+        # Populate inboxes and outboxes with messages between the user and other users.
+        conv = ConversationFactory(primary_user=self.user, number_of_other_users=2)
+        assert self.user.is_active
```

Comment:
Let's use the assert methods from unittest.

Reply:
This applies to all new/edited tests that are not using the the assert methods from the `unittest` framework.